Upgrade clubhouse lib

j-martin opened the pull request:
Also add editorconfig file.

andjosh commented:
Thanks for bumping this version! 

What's the reason to also include `node-fetch`?

j-martin commented:
I am pretty sure I added this while debugging the search endpoint API. Which let to this https://github.com/clubhouse/clubhouse-lib/pull/30 https://github.com/clubhouse/clubhouse-lib/pull/32 and finally https://github.com/clubhouse/clubhouse-lib/pull/31

I removed it from the PR.
